setup.py

```diff
@@ -7,7 +7,7 @@
 
 MAJOR = 0
 MINOR = 2
-MICRO = 1
+MICRO = 2
 
 IS_RELEASED = False
```

Comment:
I'm confused. You mean this tagged commit is not what's released? You release it with this flag flipped and then flip it back?